org.hlwd.biblemultithelife.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.hlwd.biblemultithelife
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

Comment:
24.08 is the current one

Reply:
OK I thought it was not stable.

Reply:
what piece of information made you think this? So I can get is addressed.

Reply:
Updated to 24.08